content/terms/security.md

```diff
@@ -12,7 +12,7 @@ If you have specific questions or concerns, contact us at security@sourcegraph.c
 If you think you have discovered a security vulnerability in our product, please follow our instructions on [how to report a security vulnerability](https://handbook.sourcegraph.com/product-engineering/engineering/cloud/security/reporting-vulnerabilities).
 
 For details on our information security practices or to request a copy of our SOC 2 Report, please visit our 
-<Badge text="Security Trust Portal" color="blue" size="large" onClick={() => window.open('https://security.sourcegraph.com/', '_blank', 'noopener')} />.
+<a href="https://security.sourcegraph.com/" target="_blank" rel="noopener"><Badge text="Security Trust Portal" color="blue" size="large" /></a>.
```

Comment:
I'm noticing the hover states aren't applied because we're no longer using `onClick`. I think one of our global styles is applying an underline to the badge text on hover since we're using the `a` element here.

Reply:
Sweet, thanks Becca! Could we rename the prop to `link` instead of `href`? Other than that, looks great!